json-endpoints/jetpack/class-jetpack-json-api-install-backup-helper-script-endpoint.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * API endpoint /sites/%s/install-backup-helper-script
+ * This API endpoint installs a Helper Script to assist Jetpack Backup fetch data
+ *
+ * @package Jetpack
+ */
+
+use Automattic\Jetpack\Backup\Helper_Script_Manager;
+
+class Jetpack_JSON_API_Install_Backup_Helper_Script_Endpoint extends Jetpack_JSON_API_Endpoint { 
```

Comment:
Have you considered creating those endpoints as v2 endpoints (like [this one](https://github.com/Automattic/jetpack/blob/3ed3de86a58730377bbc29f8b7df7aa7729f28ce/_inc/lib/core-api/wpcom-endpoints/memberships.php))? I thought we try to create new endpoints with this mechanism, rather than the legacy v1 one.

Just in case it's helpful, this post is a good place to read more about the different types of endpoints: PCYsg-aqU-p2